app/templates/createclasssuccess.html

```diff
@@ -0,0 +1,18 @@
+﻿{% extends "base.html" %}
+{% block style %}
+<link rel="stylesheet" href="../static/content/createstyle.css">
+{% endblock %}
+{% block content %}
+{% include "createclass.html" %}
```

Comment:
This does not address the `method="post"` issue. Still not sure why there's a form on this page.